lib/stemcell/launcher.rb

```diff
@@ -232,14 +243,14 @@ def kill(instances, opts={})
 
       errors = run_batch_operation(instances) do |instance|
         begin
-          @log.warn "Terminating instance #{instance.id}"
-          instance.terminate
+          @log.warn "Terminating instance #{instance.instance_id}"
+          ec2.terminate_instances(instance_ids: [instance.instance_id])
```

Comment:
Since `Instance::terminate` is no longer used, a single `terminate_instances` can be made to terminate all instances. Errors still need to be checked though. Also this will allow for the removal of `run_batch_operations`, which should no longer be needed now that the SDK is able to retry operations.

Reply:
That was my original intention. The reason I didn't change it is because it changes the behavior of stemcell. That is, terminate_instances is transactional -- they either all terminate or none do. If you're ok with that behavior I'll update it to terminate them all in one call.

Reply:
To be more exact, it seems it doesn't terminate these in the same AZ. In any case, we can just remove the failed ones and retry since the operation is idempotent. Less clunky than the current approach and we use this pattern in other places as well.